src/cli/near.rs

```diff
@@ -122,6 +182,89 @@ pub enum WriteCommand {
     FactoryUpdate {
         wasm_bytes_path: String,
     },
+    // deploy_code
+    DeployCode {
+        code_byte_hex: String,
+    },
+    // call
+    Call {
+        call_byte_hex: String,
+    },
+    // register_relayer
+    RegisterRelayer {
+        relayer_eth_address_hex: String,
+    },
+    // ft_on_transfer
+    FTOnTransfer {
+        sender_near_id: String,
+        amount: String,
+        msg: String,
+    },
+    // deploy_erc20_token
+    DeployERC20Token {
+        nep141: String,
+    },
+    // begin_chain
+    BeginChain {
+        chain_id: String,
+        genesis_alloc: String,
+    },
+    // begin_block
+    BeginBlock {
+        /// The current block's hash (for replayer use).
+        hash: String,
+        /// The current block's beneficiary address.
+        coinbase: String,
+        /// The current block's timestamp (in seconds since the Unix epoch).
+        timestamp: String,
+        /// The current block's number (the genesis block is number zero).
+        number: String,
+        /// The current block's difficulty.
+        difficulty: String,
+        /// The current block's gas limit.
+        gaslimit: String,
+    },
+    // withdraw
+    Withdraw {
+        recipient_address: String,
+        amount: String,
+    },
+    /* 
+    // deposit
+    Deposit {
+        raw_proof: String,
+    },
+    */
```

Comment:
Out of curiosity, why is this one commented out? I'm not necessarily saying we _should_ have it in the CLI because it would be quite difficult to use, but I wondered what your reasoning was. If we decide not to include it then I think it's ok to simply delete this code instead of having it commented out.

Reply:
It has been commented out because there was a specific error. I am happy to share with you.